plugin-dev/Source/Sentry/Private/Android/SentrySubsystemAndroid.h

```diff
@@ -18,6 +18,9 @@ class SentrySubsystemAndroid : public ISentrySubsystem
 	virtual USentryId* CaptureMessageWithScope(const FString& message, const FConfigureScopeNativeDelegate& onConfigureScope, ESentryLevel level) override;
 	virtual USentryId* CaptureEvent(USentryEvent* event) override;
 	virtual USentryId* CaptureEventWithScope(USentryEvent* event, const FConfigureScopeNativeDelegate& onConfigureScope) override;
+	virtual USentryId* CaptureException(const FString& type, const FString& message, int32 framesToSkip) override;
```

Comment:
That's a public API, right? Should we really require `framesToSkip` from users?

Reply:
No, we have `CaptureException` for internal use only to report asserts/ensures.